data/korath/korath havens.txt

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) 2022 by Saugia & Terin
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+fleet "Korsmanath A'awojs"
+	government "Korath (Civilian)"
+	names "korath"
+	fighters "korath"
+	cargo 3
+	variant 21
+		"Korsmanath A'awoj"
+		"Korath Chaser" 14
+		"Korath Chaser (Expeller)" 10
+	variant 9
+		"Korsmanath A'awoj (Strike)"
+		"Korath Chaser" 17
+		"Korath Chaser (Expeller)" 7
+	variant 6
+		"Korsmanath A'awoj (Special)"
+		"Korath Chaser (Expeller)" 13
+		"Korath Chaser (Digger)" 6
+		"Korath Chaser" 5
+	variant
+		"Korsmanath A'awoj (Blaze)"
+		"Korath Chaser (Expeller)" 15
+		"Korath Chaser" 9
+
+mission "North Korsmanath A'awojs (Natural)"
+	invisible
+	landing
+	repeat
+	to offer
+		random < 50
+		not "North Korsmanath A'awojs (Natural): active"
```

Comment:
```suggestion
		not "North Korsmanath A'awojs (Natural): active"
		or
			random < 50
			not "North Korsmanath A'awojs (Staying): active"
```
This, with the other change suggested, would guarantee that there is always a haven north.